File: Calculators/Calculator.java
public class Calculator {

    static int sum(int a, int b) {
        return a + b;
    }

    static int subtraction(int a, int b) {
        return a - b;
    }

    public static void main(String[] args) {
        int num1 = 10;
        int num2 = 4;

        System.out.println("=== ABM Calculator (Java) ===");
        System.out.println("Number 1: " + num1);
        System.out.println("Number 2: " + num2);
        System.out.println("Sum:         " + num1 + " + " + num2 + " = " + sum(num1, num2));
        System.out.println("Subtraction: " + num1 + " - " + num2 + " = " + subtraction(num1, num2));
    }
}

File: C#/Calculator.cs
using System;

class Calculator
{
    static int Sum(int a, int b)
    {
        return a + b;
    }

    static int Subtraction(int a, int b)
    {
        return a - b;
    }

    static void Main(string[] args)
    {
        int num1 = 10;
        int num2 = 4;

        Console.WriteLine("=== ABM Calculator (C#) ===");
        Console.WriteLine($"Number 1: {num1}");
        Console.WriteLine($"Number 2: {num2}");
        Console.WriteLine($"Sum:         {num1} + {num2} = {Sum(num1, num2)}");
        Console.WriteLine($"Subtraction: {num1} - {num2} = {Subtraction(num1, num2)}");
    }
}
